examples/docs_snippets/docs_snippets/guides/dagster/development_to_production/branch_deployments/repository_v2.py

```diff
@@ -26,11 +26,12 @@
                 "database": f"PRODUCTION_CLONE_{os.getenv('DAGSTER_CLOUD_PULL_REQUEST_ID')}",
             }
         ),
-        "snowflake": snowflake_resource.configured(
-            {
-                **snowflake_config,
-                "database": f"PRODUCTION_CLONE_{os.getenv('DAGSTER_CLOUD_PULL_REQUEST_ID')}",
-            }
+        "snowflake": snowflake_resource(
```

Comment:
This IMO is the API we should shoot for

Reply:
i kinda think this would be better as `SnowflakeResource(...)`